neptune/new/internal/backends/neptune_backend_mock.py

```diff
@@ -218,27 +226,32 @@ def create_model_version(self, project_id: str, model_id: str) -> ApiExperiment:
     def create_checkpoint(self, notebook_id: str, jupyter_path: str) -> Optional[str]:
         return None
 
-    def get_project(self, project_id: str) -> Project:
-        return Project(self._project_id, self.PROJECT_NAME, self.WORKSPACE_NAME)
+    def get_project(self, project_id: QualifiedName) -> Project:
+        return Project(
+            id=self._project_id,
+            name=self.PROJECT_NAME,
+            workspace=self.WORKSPACE_NAME,
+            sys_id=self.PROJECT_KEY,
+        )
 
     def get_run(self, run_id: str) -> ApiExperiment:
         raise RunNotFound(run_id)
 
-    def get_model(self, model_id: str) -> ApiExperiment:
+    def get_model(self, model_id: QualifiedName) -> ApiExperiment:
         return ApiExperiment(
-            id=model_id,
+            id=UniqueId(str(uuid.uuid4())),
```

Comment:
Because it was potentially wrong. `ApiExperiment.id` is `UniqueId`, and `model_id` in `get_model` can be either `UniqueId` (as used in sync) or `QualifiedName` (as used in `init_run`).